devel/glib20/pkg-plist

```diff
@@ -332,28 +332,28 @@ share/gettext/its/gschema.its
 share/gettext/its/gschema.loc
 %%DATADIR%%-2.0/codegen/__init__.py
 %%DATADIR%%-2.0/codegen/__init__.pyc
-%%DATADIR%%-2.0/codegen/__init__.%%PYTHON_PYOEXTENSION%%
```

Comment:
Why remove these?

Reply:
in /usr/ports/Mk/Uses/python.mk the python filename extension is automatically changed using awk.